### DISCUSSION TITLE
Firebase FCM not working in Windows, same code working in MacOS

### DISCUSSION THREAD

**Discussion by 2manoj1** [firebase/firebase-js-sdk#7572] [Q&A]
- Firebase 10.1.0

When I triger test message from firebase console using token, its not coming in OnMessage or Background event. But same code working MacOS. I am using Windows 10, and latest chrome browser.

FYI - service worker push notification test working. able to see notification. All others sites notification coming.

Please help me.

**Comment by sprabowo** [comment]
+1 having issue with Win 10, Firebase sdk js v10.0.0 and chrome browser v106

**Reply by sprabowo** [reply]
can you please check? @schmidt-sebastian

**Reply by nayakavcg** [reply]
any update on this?

**Reply by sprabowo** [reply]
i ended up downgrade to latest v9
